### PR TITLE
[DataFactory] generate SDK for tumbling window rerun trigger

### PR DESCRIPTION
**Pull request by haoyingli**
This SDK is generated for the following swagger change:
https://github.com/Azure/azure-rest-api-specs/pull/4060

## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj [hunk 1]
```diff
@@ -16,6 +16,7 @@
 		- Added support for SecureInput in activities
 		- Added ScriptActions to on demand HDI linked service
 		- Added support for recursive Cancel operation on runs
+		- Added TumblingWindowRerunTrigger API
```
Comment (dsgouda) [inline src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: version `2.3.0` is already published, is this a part of a newer release?

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj (hunk 1 above)
Comment (haoyingli) [reply src/SDKs/DataFactory/Management.DataFactory/Microsoft.Azure.Management.DataFactory.csproj]: Yes, in 2 weeks @hvermis will update the version when the team is ready to release

**Review by dsgouda — APPROVED** [review]
LGTM

File: src/SDKs/DataFactory/Management.DataFactory/changelog.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 		- Added support for SecureInput in activities
 		- Added ScriptActions to on demand HDI linked service
 		- Added support for recursive Cancel operation on runs
+		- Added TumblingWindowRerunTrigger API
```
Comment (hvermis) [inline src/SDKs/DataFactory/Management.DataFactory/changelog.md]: This should go under the section above

**Pull request merged**